src/library.js

```diff
@@ -3351,6 +3351,7 @@ LibraryManager.library = {
     return Number(str);
   },
 
+#if LINK_AS_CXX
```

Comment:
this seems surprising as libunwind is not a C++ library, it's a C API. Is it because only C++ uses it? (are we sure nothing else does?)

Reply:
No but it is a C++ support library that is excluded from the link when you build with `-nostdlib++`.   Whenever we exclude the native part we also need to exclude the JS part.